web/packages/teleterm/README.md

```diff
@@ -118,6 +118,51 @@ node-pty](https://github.com/microsoft/node-pty#dependencies).
 
 To create arm64 deb and RPM packages you need to provide `USE_SYSTEM_FPM=1` env var.
 
+### Windows
+
+A lot of our tooling assumes that you're running sh-compatible shell with some standard tools like
+`make` available. On Windows, that's available through Git Bash from [Git for Windows](https://gitforwindows.org/).
+
+`make build/tsh` doesn't work on Windows anyway. But you can run a simplified version of what that
+Make target calls underneath.
+
+```
+GOOS=windows CGO_ENABLED=1 go build -tags "  " -o build/tsh.exe  -ldflags '-w -s' -buildvcs=false ./tool/tsh
+```
+
+It's important for the executable to end with `.exe`. If that command doesn't work, you can always
+inspect what we currently do in [our Windows build pipeline scripts](https://github.com/gravitational/teleport/blob/983017b23f65e49350615bfbbe52b7f1080ea7b9/build.assets/windows/build.ps1#L377).
+
+#### Native dependencies on Windows
+
+On Windows, you need to pay special attention to [the dev tools needed by node-pty](https://github.com/microsoft/node-pty?tab=readme-ov-file#windows),
+especially the Spectre-mitigated libraries installed through Visual Studio Installer that are kind
+of tricky to install. If you're on an arm64 VM of Windows, you'll likely need both arm64 and x64
+versions of Spectre-mitigated libraries. This is because during `pnpm install` pnpm will try to
+build arm64 version of node-pty (which will be used for `pnpm start-term`), and during `pnpm
+package-term` it might attempt to compile x64 version of node-pty.
+
+At the time of writing, we found the following set of individual components for Visual Studio 2022
+to work with Connect build process:
+
+- MSVC v143 - VS 2022 C++ ARM build tools (Latest)
+- MSVC v143 - VS 2022 C++ ARM Spectre-mitigated libs (Latest)
```

Comment:
These two are likely not needed, but I refuse to uninstall them in case I break my setup. 😅 I assume ARM means the 32-bit architecture. On top of that, selecting Spectre-mitigated libs for installation should automatically select relevant build tools, so it's not necessary to select them separately.

@Joerger Could you check if you run into any problems if you don't install these ARM items?

@nklaassen Which individual components do you have selected in Visual Studio Installer if any?